fixtures/infoServer.json

```diff
@@ -0,0 +1,34 @@
+{
```

Comment:
Aieeee! JSON is horrible to work with.

Please make these files `.ts`, so we can have linting, auto-formatting, type-checking, and all the other goodies we expect. You can still do `export default` if you want it to work the same way from the outside.

The reason we started using JSON in the first place was because Eliran didn't know how to configure Mocha properly. Since he didn't know how to ignore specific files, he used the JSON extension as a way to ignore setup files.

You aren't using Mocha, so this problem isn't relevant. Even when we use Mocha, we always tell it to run `tests/**/*.test.ts`, so Mocha will only run files that end with ".test.ts", and ignore the rest. That means you can use the ".ts" file extension freely, and don't need to use the JSON extension as a workaround.

I see that these files are being uploaded to CouchDB. That's fine. We can still have nice things on the developer's machine.

Reply:
These files are defaults primarily so that we have a template for how to create the syntax on CouchDB. I kept them as JSON as it's actually easier to copy and paste to/from Couch and the JSON files.